<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 40, 51);</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>541</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(197, 198, 199);
font: 75 9pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(69, 162, 158);
font: 75 12pt &quot;Terminal&quot;;</string>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
